public/input_text_load_data.js

```diff
@@ -119,6 +146,43 @@ function loadData(collection) {
   });
 }
 
+// Changes XML to JSON
```

Comment:
Right now our project only works for JSON documents and Dave wants functionality with XML... Fei's first instinct was a quick conversion to JSON. Should I look to keep the XML as is?